website/source/docs/agent/configuration/vault.html.md

```diff
@@ -51,20 +51,20 @@ vault {
 - `task_token_ttl` `(string: "")` - Specifies the TTL of created tokens when
```

Comment:
Use:
Specifies if users submitting jobs to the Nomad server should be required to provide their own Vault token, proving they have access to the policies listed in the job. This option should be disabled in an untrusted environment.